packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -251,6 +251,7 @@ export const ensureBookingInputsHaveSystemFields = ({
       type: "multiemail",
       editable: "system-but-optional",
       name: "guests",
+      defaultPlaceholder: "enter_address",
```

Comment:
why did you add this ?

Reply:
I removed it can you please check now..
Thank You!